src/App.jsx

```diff
@@ -1,35 +1,23 @@
-import { useState } from 'react';
-import reactLogo from './assets/react.svg';
-import viteLogo from '/vite.svg';
 import './App.css';
+import Dog from './components/Dog';
+import DogList from './components/DogList';
 
-function App() {
-  const [count, setCount] = useState(0);
-
+const App = () => {
   return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank" rel="noreferrer">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank" rel="noreferrer">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
+    <div className="App">
+      <header className="App-header">
+        <h1>Flasky</h1>
+      </header>
+      <main>
+        <div>
+          <DogList />
+          <Dog />
+          <Dog />
+          <Dog />
```

Comment:
Can these be moved to being within DogList? In the [Student App](https://github.com/AdaGold/student-app/blob/01-react-components/src/components/StudentList.jsx) we start with hard-coding children, but they're within the StudentList, not siblings to it.

Reply:
Can definitely move these inside of `DogList`, thanks for the reference!